.eleventy.js

```diff
@@ -97,15 +97,15 @@ const filenameToData = (f) => ({
     .map(({ name, id }) => ({
       name,
       id,
-      safeScore: getSafeScore(filterByTerritorioId(all, id)),
+      safeScore: Number(getSafeScore(filterByTerritorioId(all, id))),
```

Comment:
Muchísimas gracias, @albfernandez. Tiene buena pinta la PR, hago alguna prueba más cuando saque un rato y la mergeo si está todo ok.

A priori creo que si getSafeScore ya devuelve un Number, no hace falta volverlo a convertir aquí y en la L108.

```suggestion
      safeScore: getSafeScore(filterByTerritorioId(all, id)),
```